lib/library_web/controllers/page_controller.ex

```diff
@@ -3,6 +3,8 @@ defmodule LibraryWeb.PageController do
   import Library.Books
   alias Library.GoogleBooks
 
+  plug LibraryWeb.Plugs.RequireAuth when action in [:checkout, :checkin]
```

Comment:
We can add this to the router as well for all routes that require auth. Then if it's necessary in several controllers, we aren't repeating a similar line in each controller where it's required.